docs/sdk/tutorials/plugins_example.md

```diff
@@ -201,6 +237,31 @@ kili.activate_plugin_on_project(plugin_name, project_id=project_id);
     Plugin with name "Plugin bbox count" activated on project "clcysbp9o2d7w0krfghuf7le1"
 
 
+## Step 4 bis: Upload the plugin from a .py file
+
+Alternatively, you can also create a plugin directly from a `.py` file.
+
+- The upload will create the necessary builds to execute the plugin (it will take a few minutes)
+- After the activation, you can start using your plugin right away.
+
+
+```python
+path_to_plugin = "main.py"
+plugin_name = "Plugin bbox count"
+
+from kili.exceptions import GraphQLError
+
+try:
+    kili.upload_plugin(path_to_plugin, plugin_name)
+except GraphQLError as error:
+    print(str(error))
+```
+
+
+```python
+kili.activate_plugin_on_project(plugin_name, project_id=project_id);
```

Comment:
I think it's better to keep it that way so that users understand clearly that the first block is to create a plugin and the second one is to activate it

Reply:
Nevermind, I thought it was in the same cell as the `upload_plugin`. You're right !